Tests/FeatherDatabaseTests/_Utils/XCTAssertThrowsAsync.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  File.swift
+//
+//
+//  Created by Tibor Bodecs on 29/02/2024.
+//
+
+import XCTest
+
+func XCTAssertThrowsAsync<E: Error>(
+    _ expression: () async throws -> Void,
+    _: E.Type,
+    _ errorBlock: ((E) async throws -> Void)? = nil,
+    _ message: String,
```

Comment:
```suggestion
    _ message: @autoclosure () -> String,
```
(to match XCTest's behavior)